src/frontend/get-jira-configuration.ts

```diff
@@ -64,6 +64,12 @@ export default async (
 	try {
 		const jiraHost = req.session.jiraHost;
 
+		if(!jiraHost) {
+			req.log.warn({jiraHost, req, res}, "Missing jiraHost");
+			res.status(404).send(`Missing Jira Host '${jiraHost}'`);
+			return;
+		}
```

Comment:
Adding `jiraHost` in the warn log event and the response doesn't make much sense when it's `undefined` anyways, does it?

Reply:
LOL. good find. that was stupid...